cpp/src/sampling/detail/sampling_utils_sg.cu

```diff
@@ -84,6 +85,7 @@ template rmm::device_uvector<int64_t> get_global_adjacency_offset(
   graph_view_t<int64_t, int64_t, double, false, false> const& graph_view,
   rmm::device_uvector<int64_t> const& global_degree_offsets,
   rmm::device_uvector<int64_t> const& global_out_degrees);
+#endif
```

Comment:
Better just delete?

Reply:
And can I just close PR https://github.com/rapidsai/cugraph/pull/2370?

Reply:
I think you can just close PR 2370.

I deleted both of these blocks of code (in _sg and _mg).  Thanks for catching that.